pandas/io/formats/format.py

```diff
@@ -533,6 +533,7 @@ def __init__(
         frame: "DataFrame",
         columns: Optional[Sequence[str]] = None,
         col_space: Optional[Union[str, int]] = None,
+        col_space_cols: Optional[Sequence[str]] = None,
```

Comment:
rather than adding an additional argument, can you make the behavior similar to the `formatters` argument. (and reuse code would be great). i.e. also accept a list or a dict. (or Sequence or Mapping)

if a list, the number of elements in the list should be the same as the number of columns.

Reply:
Hi @simonjayhawkins , Sorry I didn't get your comment. The following statement seems confusing to me.
> rather than adding an additional argument, can you make the behavior similar to the formatters argument

Did you mean to NOT add the additional argument `col_space_cols` and use existing `formatters` argument for this change or add an argument similar to `formatters`, which accepts a list or a dict.

Also, 
> if a list, the number of elements in the list should be the same as the number of columns

Not sure if I understand this, if we want to change width of only a single column, we can pass only that column's name in the list, right ?

Please let me know if I missing something here.

Reply:
Hi @simonjayhawkins , did you get a chance to look at the my concern ? I think I need to make the new argument similar to the `formatters` argument, but would still wait for your confirmation/opinion. Thanks!